sinks/src/main/resources/application.conf

```diff
@@ -1,4 +1,8 @@
 "snowplow": {
+  "output" {
+    "type": "File"
+    "path": "file:path/to/my/file"
```

Comment:
The `application.conf` file is supposed to contain sensible defaults.  If there is no sensible default, then it is better to remove that value from `application.conf`.  So we force the user to supply a value at runtime, or else their config is not valid.

We normally "document" which params to do not have defaults by supplying a minimal example config file, [e.g. this one for enrich](https://github.com/snowplow/enrich/blob/master/config/config.kinesis.minimal.hocon).  I'm not saying you need to do this, it's just a FYI.

But I do think `path` either needs a more sensible default, or should be removed here.

Reply:
Ah I knew that already I just confused myself. Will circle back round and fix this afternoon, ty!